internal/exec/helmfile.go

```diff
@@ -160,14 +160,31 @@ func ExecuteHelmfile(cmd *cobra.Command, args []string) error {
 	envVars := append(info.ComponentEnvList, []string{
 		fmt.Sprintf("AWS_PROFILE=%s", helmAwsProfile),
 		fmt.Sprintf("KUBECONFIG=%s", kubeconfigPath),
-		fmt.Sprintf("NAMESPACE=%s", context.Namespace),
-		fmt.Sprintf("TENANT=%s", context.Tenant),
-		fmt.Sprintf("ENVIRONMENT=%s", context.Environment),
-		fmt.Sprintf("STAGE=%s", context.Stage),
-		fmt.Sprintf("REGION=%s", context.Region),
 		fmt.Sprintf("STACK=%s", info.Stack),
 	}...)
 
+	// Append the context ENV vars (first check if they are not set by the caller)
+	env := os.Getenv("NAMESPACE")
```

Comment:
can we make these `ATMOS_` specific inputs

e.g.

```suggestion
	env := os.Getenv("ATMOS_NAMESPACE")
```

or

```suggestion
	env := os.Getenv("ATMOS_CONTEXT_NAMESPACE")
```